frontend/src/fixtures/personalSchedulesFixtures.js

```diff
@@ -1,28 +1,63 @@
 const personalSchedulesFixtures = {
-    onePersonalSchedule: {
-        "id": 1,
-        "name": "TestName",
-        "description": "TestDescription",
-        "quarter": "W08"
-    },
+    onePersonalSchedule: [
+        {
+            "id": 1,
+            "user": {
+                "id": 1,
+                "email": "phtcon@ucsb.edu",
+                "googleSub": "115856948234298493496",
+                "pictureUrl": "https://lh3.googleusercontent.com/-bQynVrzVIrU/AAAAAAAAAAI/AAAAAAAAAAA/AMZuucmkGuVsELD1ZeV5iDUAUfe6_K-p8w/s96-c/photo.jpg",
+                "fullName": "Phill Conrad",
+                "givenName": "Phill",
+                "familyName": "Conrad",
+                "emailVerified": true,
+                "locale": "en",
+                "hostedDomain": "ucsb.edu",
+                "admin": true
+            },
+            "description": "Advanced Applications Programming",
+            "quarter": "20221",
+            "name": "CS156"
```

Comment:
So, these fixtures give the idea that a personal schedule corresponds to a course, which could be confusing for future developers.  A personal schedule does NOT correspond to a specific course, but rather to a group of courses, like your schedule for all the courses you are taking in W22 or S22, or the courses you WANT to take if you get into them, etc.